flepimop/gempyor_pkg/src/gempyor/model_info.py

```diff
@@ -170,18 +169,18 @@ def __init__(
                         "modifiers"
                     ].get()
                     raise ValueError(
-                        "Not implemented yet"
+                        "This feature has not been implemented yet."
                     )  # TODO create a Stacked from all
             elif self.seir_modifiers_scenario is not None:
                 raise ValueError(
-                    "An seir modifiers scenario was provided to ModelInfo but no 'seir_modifiers' sections in config"
+                    "A seir modifiers scenario was provided to `ModelInfo` but there is no `seir_modifiers` sections in config."
                 )
             else:
-                logging.info("Running ModelInfo with seir but without SEIR Modifiers")
+                logging.info("Running `ModelInfo` with seir but without SEIR Modifiers")
 
         elif self.seir_modifiers_scenario is not None:
             raise ValueError(
-                "A seir modifiers scenario was provided to ModelInfo but no 'seir:' sections in config"
+                "A seir modifiers scenario was provided to `ModelInfo` but there is no `seir_modifiers` sections in config."
```

Comment:
I would have to take a closer look at the code to confirm, but the 'seir' and 'seir_modifiers' sections are distinct. Is it correct to change the section name here?

Reply:
Mm good catch, confirming with Joseph but you're probably right.

Reply:
Okay, as I now understand, it should say '...no `seir_modifiers` sections...', because the greater `if` statement of this error is only entered if there is a `seir:` section. So, it isn't correct to say that there is no `seir:` section in the error message about modifiers.